recipe/meta.yaml

```diff
@@ -161,6 +176,232 @@ outputs:
         - test -f $PREFIX/include/mkl.h  # [unix]
         - if not exist %LIBRARY_INC%\\mkl.h exit 1  # [win]
 
+  - name: onemkl-sycl-blas
+    version: {{ mkl_version }}
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    number: {{ mkl_buildnum|int + dstbuildnum|int }}
+    requirements:
+      run:
+        - {{ pin_subpackage('mkl', exact=True) }}
+        - dpcpp-cpp-rt {{ mkl_version.split('.')[0] }}.*
+        - intel-opencl-rt {{ mkl_version.split('.')[0] }}.*
+    about:
+      home: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
+      license: LicenseRef-ProprietaryIntel
+      license_family: Proprietary
+      license_file:
+         - onemkl-sycl-blas/info/licenses/license.txt
+         - onemkl-sycl-blas/info/licenses/tpp.txt
+      summary: Intel® oneAPI Math Kernel Library runtime libraries
+      description: |
+        Intel® oneAPI Math Kernel Library is Intel®-Optimized Math Library for Numerical Computing on CPUs & GPUs
+        This package is a repackaged set of binaries obtained directly from Intel\'s anaconda.org channel.
+      doc_url: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
+    test:
+      commands:
+          - ls -A $PREFIX/lib/*  # [unix]
+          - test -f $PREFIX/lib/libmkl_sycl_blas.so  # [unix]
+          - if not exist %LIBRARY_BIN%\\mkl_sycl_blas.4.dll exit 1  # [win]
+
+  - name: onemkl-sycl-datafitting
+    version: {{ mkl_version }}
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    number: {{ mkl_buildnum|int + dstbuildnum|int }}
+    requirements:
+      run:
+        - {{ pin_subpackage('mkl', exact=True) }}
+        - dpcpp-cpp-rt {{ mkl_version.split('.')[0] }}.*
+        - intel-opencl-rt {{ mkl_version.split('.')[0] }}.*
+    about:
+      home: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
+      license: LicenseRef-ProprietaryIntel
+      license_family: Proprietary
+      license_file:
+         - onemkl-sycl-datafitting/info/licenses/license.txt
+         - onemkl-sycl-datafitting/info/licenses/tpp.txt
+      summary: Intel® oneAPI Math Kernel Library runtime libraries
+      description: |
+        Intel® oneAPI Math Kernel Library is Intel®-Optimized Math Library for Numerical Computing on CPUs & GPUs
+        This package is a repackaged set of binaries obtained directly from Intel\'s anaconda.org channel.
+      doc_url: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
+    test:
+      commands:
+          - ls -A $PREFIX/lib/*  # [unix]
+          - test -f $PREFIX/lib/libmkl_sycl_data_fitting.so  # [unix]
```

Comment:
Since we already use SONAME version in the filename for Windows, perhaps we should be changing for `libmkl_sycl_data_filling.so.4` here too